Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM node:18-alpine3.16 AS ninja-node
```

Comment:
Please provide two dockerfiles, one for development, one for production. Into production mode we want to build an image with already transpiled code from TS into pure JS.